rts/lra/coordinator-quarkus/pom.xml

```diff
@@ -13,11 +13,16 @@
 
     <properties>
         <narayana.version>${project.version}</narayana.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
@mmusgrov FYI I note we have this in a few places but different expectations: https://github.com/jbosstm/narayana/search?p=2&q=%22maven.compiler.target%22 - perhaps we would like to unify this?